pom.xml

```diff
@@ -152,13 +152,23 @@
           <groupId>org.codehaus.plexus</groupId>
           <artifactId>plexus-utils</artifactId>
         </exclusion>
+	<exclusion>
+	  <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
+	</exclusion>
       </exclusions>
     </dependency>
     <dependency>
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-utils</artifactId>
       <version>3.0.16</version>
     </dependency>
+    <dependency>
```

Comment:
submodule extends parent's dependency, maybe you can remove the junit in submodules or move this junit to from `dependency` to `dependencyManagement` (version management, same version in one pom)

Reply:
updated.

Reply:
remove version in submodule